tests/integration/cmd_analyze_test.go

```diff
@@ -40,6 +40,8 @@ var _ = Describe("odo analyze command tests", func() {
 					Expect(helper.IsJSON(stdout)).To(BeTrue())
 					helper.JsonPathContentIs(stdout, "0.devfile", "nodejs")
 					helper.JsonPathContentIs(stdout, "0.devfileRegistry", "DefaultDevfileRegistry")
```

Comment:
While you are at it, can you also update this check? It was failing on my local environment because `odo` was using my in-cluster devfile registry.
I think we could relax this with:

```suggestion
					helper.JsonPathExist(stdout, "0.devfileRegistry")
```